_episodes_rmd/14-tidyr.Rmd

```diff
@@ -212,10 +210,7 @@ gap_long$year <- as.integer(gap_long$year)
 
 ## From long to intermediate format with pivot_wider()
 
-It is always good to check work. So, let's use the opposite of `pivot_longer()` to
-'widen' our observation variables back out with the aptly named `pivot_wider()`. We
-can then pivot our `gap_long()` to the original intermediate format or the
-widest format. Let's start with the intermediate format.
+It is always good to check work. So, let's use the second `pivot` function, `pivot_wider()`, to 'widen' our observation variables back out.  `pivot_wider()` is the opposite of `pivot_longer()`, making a dataset wider by increasing the number of columns and decreasing the number of rows. We can use `pivot_wider()` to pivot or reshape our `gap_long()` to the original intermediate format or the widest format. Let's start with the intermediate format.
```

Comment:
I don't think `gap_long` is a function, so `gap_long()` should be replaced with `gap_long`.